jupyterhub/tests/mockservice.py

```diff
@@ -1,44 +1,49 @@
-"""Mock service for testing
+"""Helpers for testing a Service
```

Comment:
Thanks @minrk for both the clarification and the suggested docstring. Understanding other folks' code is hard. Hoping this will help the next devs. Updating...